### ISSUE TITLE
Present what NixOS can practically do

### ISSUE THREAD

**Issue by davidak** [NixOS/nixos-homepage#362]
We could have a section on the landing page with examples what people use NixOS for. This is an opportunity to show that this is a community project with lot of innovative and cool possibilities, but also used seriously as a stable foundation for companies.

So it's related to https://github.com/NixOS/nixos-homepage/issues/284.

For example:

- LumiGuide develops and maintains bicycle and car parking guidance systems that are deployed in various cities in the Netherlands and abroad.
- i hear often that Haskell developers like NixOS. what are practical examples where NixOS really helps? when we present this on the landing page, more Haskell developers will come

Related: https://nixos.wiki/wiki/Nix_Community

Nix Ecosystem users and contributors are a mixed bunch, ranging from hobbyist to professional, from industry to academia. Typical user groups include:

- desktop users using Nix and NixOS to make life on their personal machines easier;
- power users duplicating their environment across their personal and work machines;
- developers using Nix to reproducibly build their software, and to manage various development environments;
- system administrators declaratively configuring servers and clusters with NixOS and NixOps;
- researchers interested in making the computation in their research reproducible.

let's show examples from all groups and present this to the visitors

**Comment by garbas** [comment]
I removed the reference to Mozilla since it is not using it anymore.

**Comment by davidak** [comment]
@garbas can you clarify this more? [nixpkgs-mozilla](https://github.com/mozilla/nixpkgs-mozilla) has the last commit in february and there are still new issues and pull requests. if it is no longer maintained, that should be mentioned in the readme and the project archived! We link to that in https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/rust.section.md#using-the-rust-nightlies-overlay.

**Comment by davidak** [comment]
I don't believe that Mozilla is not using Nix anymore. The last commit was 22 days ago and the latest pull request 7 days ago.

@nbp @rail can you tell me **how** Mozilla uses Nix and NixOS? We might publish that on the website.

**Comment by rail** [comment]
I'm not aware of anything other than personal usage, we have a few people actively using NixOS as their main OS. We used to use Nix to deploy various services, but not anymore. :/

**Comment by jonringer** [comment]
@rail I'm curious as why nix was discontinued :)

**Comment by rail** [comment]
Mostly because we had not enough expertise in our team. 1.5 person in a 15-person team wasn't sustainable.

**Comment by jonringer** [comment]
that sounds reasonable, was just curious. Thanks! :)
